.gitignore

```diff
@@ -37,6 +37,7 @@ build-iPhoneSimulator/
 /_yardoc/
 /doc/
 /rdoc/
+.adr-dir
```

Comment:
We actually check this in to the repo as it defines which folder to generate ADRs in. If we don't all have the same config in this file, new ADRs will end up in weird places.

Reply:
Will do. I excluded it because I assumed not everyone was using `adr-tools`, but it makes sense to include it for those that do.